app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java

```diff
@@ -103,6 +103,7 @@
     private Place place;
     private List<UploadableFile> uploadableFiles = Collections.emptyList();
     private int currentSelectedPosition = 0;
+    private boolean isMultipleFilesSelected = false;
```

Comment:
Ah, I forgot! We need javadoc for each new method/member.
After that I will merge.

Reply:
@nicolas-raoul Should we add Javadoc here or getter fields?

Reply:
Both methods **and** members, actually. Sorry for the hassle, but documentation is usually written once, read many times, so it is a good investment :-)

Reply:
Javadoc must use this format for members too:
```
/**
 * Example javadoc
 */
```
Thanks!